### PR TITLE
Parse incoming responses into Message structs.

### PR DESCRIPTION
**Pull request by MattWindsor91**
This is currently entirely pointless, but hey, it looks purdy.

### CODE REVIEW

File: connector.go [hunk 1]
```diff
@@ -93,21 +94,58 @@ func (c *Connector) Run() {
 	}
 }
 
+// lineToMessage constructs a Message struct from a line of word-strings.
+func lineToMessage(line []string) (msg *baps3protocol.Message, err error) {
+	if len(line) == 0 {
+		err = fmt.Errorf("cannot construct message from zero words")
+	} else {
+		msg = baps3protocol.NewMessage(baps3protocol.LookupWord(line[0]))
+		for _, word := range line[1:] {
```
Comment (wlcx) [inline connector.go]: using word here is a little confusing w.r.t the context of command words above

**Comment by wlcx** [comment]
Also Test for message.Arg() perhaps

File: connector.go (hunk 1 above)
Comment (MattWindsor91) [inline connector.go]: Indeed it is.  It should be `arg`.

**Comment by wlcx** [comment]
Rebase plz

**Comment by wlcx** [comment]
Sweetness

**Pull request merged**